prow/flagutil/github.go

```diff
@@ -49,7 +50,9 @@ func (o *GitHubOptions) AddFlagsWithoutDefaultGithubTokenPath(fs *flag.FlagSet)
 
 func (o *GitHubOptions) addFlags(wantDefaultGithubTokenPath bool, fs *flag.FlagSet) {
 	o.endpoint = NewStrings("https://api.github.com")
+	o.gitEndpoint = "https://github.com"
```

Comment:
Just put the string literal into the `StringVar()` call (the third arg is the default value), like we do for github-token-path.

We have an `o.endpoint = NewStrings()` because `Var()` doesn't include a default value param.

Reply:
Makes sense... done.